task/task_smoke_test.go

```diff
@@ -40,9 +89,17 @@ func TestTaskSmoke(t *testing.T) {
 	}
 
 	if testName == "" {
-		testName = "smoke test"
+		testName = fmt.Sprintf("smoke test: %s", targetName)
```

Comment:
Note that `testName` is fully deterministic on purpose, so we don't inadvertently litter our test cloud accounts with flotsam resources uniwittingly.